lib/fluent/plugin/in_windows_eventlog2.rb

```diff
@@ -66,9 +67,17 @@ def initalize
     def configure(conf)
       super
       @chs = []
+      @all_chs = Winevt::EventLog::Channel.new
+      @all_chs.force_enumerate = false
 
+      if @read_all_channels
+        @all_chs.each do |all_chs|
+          @chs.push(all_chs)
+        end
```

Comment:
```suggestion
        @all_chs.map {|ch| ch.strip.downcase }.each do |uch|
          @chs.push([uch, @read_existing_events])
        end
```

Could you unify `@ch.push` style?

Reply:
I've fixed it in https://github.com/fluent/fluent-plugin-windows-eventlog/pull/48/commits/93e48ed769bc85ee63b28cc73646fd9f58c0f9a7.